crypto.c

```diff
@@ -385,10 +385,20 @@ static void dtls_ec_key_from_uint32(const uint32_t *key, size_t key_size,
   }
 }
 
-int dtls_ec_key_from_uint32_asn1(const uint32_t *key, size_t key_size,
+/* Build the EC KEY as a ASN.1 positive integer */
+int dtls_ec_key_asn1_from_uint32(const uint32_t *key, size_t key_size,
```

Comment:
Why not make `buf` of type `uint8_t` as this is used throughout this function now?

Reply:
Corrected